src/scripts/selectors/index.js

```diff
@@ -1,12 +1,15 @@
 /* @flow */
 
 import VideoRecord from 'records/VideoRecords'
-import UploadRecord from 'records/UploadRecords'
 import UserRecord from 'records/UserRecords'
 import type { RootState } from 'types/ApplicationTypes'
 
 /* Videos */
-export const getVideo = (state: RootState): ?VideoRecord => state.selectedVideo
+export const getVideo = (state: RootState): ?VideoRecord => {
+  let videoId = state.selectedVideo
```

Comment:
nitpicky, but let's try to use `const` over `let` when the value will never change (maybe I can just turn on an `eslint` rule for this ;) )